src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -798,6 +805,13 @@ __meta__ = {
 "_editor_description_": "Just something to make the microbe camera look at the right place"
 }
 
+[node name="EditorArrow" type="MeshInstance" parent="."]
```

Comment:
Should be made into a scene (maybe saved in the same folder as the mesh file). Other than this it looks good.